control-plane-operator/controllers/hostedcontrolplane/ingress/router_config.template

```diff
@@ -0,0 +1,47 @@
+global
+  log stdout local0
+  maxconn 20000
+
+defaults
+  mode tcp
+  log global
+  log-format "%ci:%cp -> %fi:%fp [%t] %ft %b/%s %Tw/%Tc/%Tt %B %ts %ac/%fc/%bc/%sc/%rc %sq/%bq"
+  option  dontlognull
+  retries 3
+  timeout http-request 30s
+  timeout queue        1m
+  timeout connect      10s
+  timeout client       86400s
+  timeout server       86400s
+  timeout tunnel       86400s
+
+frontend main
+  bind :::8443 v4v6
+  tcp-request inspect-delay 5s
+  tcp-request content accept if { req_ssl_hello_type 1 }
+  {{- range .Backends }}
+  acl is_{{ .Name }} req_ssl_sni -i {{ .HostName }}
+  {{- end }}
+  {{- range .Backends }}
+  use_backend {{ .Name }} if is_{{ .Name }}
+  {{- end }}
+  {{- if .HasKubeAPI }}
+  default_backend kube_api{{- end }}
+  
+listen health_check_http_url
+  bind :::9444 v4v6
+  mode http
+  monitor-uri /haproxy_ready
+
+resolvers coredns
+  nameserver ns1 {{ .NameServerIP }}:53
+
+{{ $ns := .Namespace }}
+{{- range .Backends }}
+backend {{ .Name }}
+  server {{ .Name }} {{ .DestinationService }}.{{ $ns }}.svc.cluster.local:{{.DestinationPort}} check resolvers coredns init-addr last,libc,none
```

Comment:
Do we need the ns and .cluster.local?
Couldn't we just use the name?
If so couldn't  this actually be a static hardcoded list?